### PR TITLE
Feat: Notifications infinite scrolling

### PR DESCRIPTION
**Pull request by BrandonArgel**

### CODE REVIEW

**Comment by BrandonArgel** [comment]
@Basultobd

File: packages/notification-system/src/assets/icons.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  >
+    <path
+      d="M1 1L13.5 13.5M26 26L13.5 13.5M13.5 13.5L26 1L1 26"
+      stroke="#1572A1"
+      stroke-width="2"
+    />
+  </svg>
+);
+
+export const Spinner = () => (
```
Comment (Basultobd) [inline packages/notification-system/src/assets/icons.js:17]: Let's add one component per file, this favor the encapsulation

File: packages/notification-system/src/views/NotificationsDashboard/index.jsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      }, randomNumber());
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  // Function to get a random number and simulate a delay
+  const randomNumber = () => {
+    const number = Math.floor(Math.random() * (1000 - 500 + 1)) + 500;
+    console.log(number);
```
Comment (Basultobd) [inline packages/notification-system/src/views/NotificationsDashboard/index.jsx:31]: Try to not push console's

**Review by Basultobd — CHANGES_REQUESTED** [review]
Could you add a GIF to see how it works? 👀

File: packages/notification-system/src/assets/icons.js (hunk 1 above)
Comment (BrandonArgel) [reply packages/notification-system/src/assets/icons.js]: Fixed

File: packages/notification-system/src/views/NotificationsDashboard/index.jsx (hunk 2 above)
Comment (BrandonArgel) [reply packages/notification-system/src/views/NotificationsDashboard/index.jsx]: Fixed

**Comment by BrandonArgel** [comment]
> Could you add a GIF to see how it works? 👀
notifications

**Review by Basultobd — APPROVED** [review]
LGTM!

**Pull request merged**